src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1546,7 +1546,7 @@ because xbuild doesn't support framework reference assemblies.
 </PropertyGroup>
 
 <Target Name="_BuildLibraryImportsCache"
-		Inputs="$(MSBuildProjectFullPath);@(ReferencePath);@(ReferenceDependencyPaths);$(_AndroidBuildPropertiesCache)"
+		Inputs="$(MSBuildProjectFullPath);@(ReferencePath);@(ReferenceDependencyPaths);$(_AndroidBuildPropertiesCache);$(_AndroidLibraryProjectImportsCache)"
```

Comment:
@alanmcgovern can you change this file locally in your system `Xamarin.Android.Common.targets`? I wasn't able to reproduce the problem anymore after I changed mine.

I feel a lot better about backporting this to `d16-0`, since it's a one-line change 😄

Reply:
Checking!

Reply:
hrmm... good news/bad news? It doesn't seem to work for me :(

Reply:
I tell a lie! It does work. I think VSMac cached the target file so it only started working when i restarted fully. Closing/reopening the project wasn't sufficient